sys/vane/dill.hoon

```diff
@@ -280,7 +281,7 @@
                   (sync %home our %base)
                 (init-sync %home our %base)
         =.  +>  ?.  ?=(?($duke $king $czar) can)  +>
-                (sync %kids our %base)
+                (show %kids):(sync %kids our %base)
```

Comment:
Can you explain why this change was necessary?

Reply:
Since all local desks are fully private by default, we need to make our `%kids` desk public explicitly if we want people to be able to sync with it.

Reply:
I think a comment here would be helpful.